src/main/scala/is/hail/methods/MinHash.scala

```diff
@@ -0,0 +1,64 @@
+package is.hail.methods
+
+import breeze.linalg.{DenseMatrix, DenseVector, min}
+import is.hail.variant.{Genotype, Variant, VariantDataset}
+import is.hail.annotations.Annotation
+import scala.collection.mutable.HashSet
+import scala.util.hashing.{MurmurHash3 => MH3}
+import scala.util.Random
+
+object MinHash {
+  // returns a k by vds.nSamples matrix
+  def kMinHash(vds: VariantDataset, k: Int): DenseMatrix[Int] = {
```

Comment:
Now that you've removed the single hash function, I'd just use minHash for this name, since there will always be a parameter `k`